website/docusaurus.config.js

```diff
@@ -0,0 +1,180 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: "Cosmos Improvement Proposals",
+  tagline: "Cosmos Improvement Proposals",
+  favicon: "img/favicon.ico",
+
+  // Set the production url of your site here
+  url: "https://your-docusaurus-test-site.com",
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: "/",
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: "Cosmos", // Usually your GitHub org/user name.
+  projectName: "CIPs", // Usually your repo name.
+
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+
+  // Even if you don't use internalization, you can use this field to set useful
+  // metadata like html lang. For example, if your site is Chinese, you may want
+  // to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en"],
+  },
+
+  presets: [
+    [
+      "classic",
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          sidebarPath: require.resolve("./sidebars.js"),
+          editUrl: "https://github.com/cosmos/CIPs/edit/master/website",
+          showLastUpdateAuthor: true,
+          showLastUpdateTime: true,
+          path: "../cips",
+          routeBasePath: "/",
+        },
+        theme: {
+          customCss: require.resolve("./src/css/custom.css"),
+        },
+      }),
+    ],
+  ],
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      // image: "img/banner.jpg",
+      docs: {
+        sidebar: {
+          autoCollapseCategories: true,
+        },
+      },
+      navbar: {
+        title: "Cosmos CIPs",
+        hideOnScroll: false,
+        logo: {
+          alt: "Cosmos CIPs Logo",
+          src: "img/logo-sdk.svg",
```

Comment:
I know the logo isn't set yet, but this should have at least another name.

Reply:
waiting on the marketing team